GOVERNANCE.md

```diff
@@ -360,12 +360,12 @@ If the pull request submitter is not a Core Member, the PR can be closed by any
 
 Astro features are discussed using a model called [Consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making). This model attempts to achieve consensus on all significant changes to Astro, but has a fallback voting procedure in place if consensus appears unattainable.
 
-**Who can vote:** All [TSC](#technical-steering-committee-tsc) members.
+A RFC async meeting is held on ad-hoc basis by Core Contributors and TSC to decided which proposals are accepted
 
 1. Anyone can submit an RFC to suggest changes to Astro.
 2. A trivial change can be discussed and approved entirely within the RFC GitHub issue, as long as there are no objections from Core or TSC members. This is not considered a formal vote.
-3. A non-trivial, significant change should be discussed within the RFC and approved during an RFC meeting call. In some cases, an RFC may be approved outside of an RFC meeting using Pull Request reviews as a proxy for votes.
-4. During an RFC meeting, the person leading the call will attempt to achieve consensus on the RFC proposal.
+3. A non-trivial, significant change should be discussed within the RFC and approved during an RFC async meeting. In some cases, an RFC may be approved outside of an RFC meeting using Pull Request reviews as a proxy for votes.
+4. During an RFC async meeting, the person leading the meeting will attempt to achieve consensus on the RFC proposal.
```

Comment:
```suggestion
3. A non-trivial, significant change should be discussed within the RFC and approved during an RFC discussion (either as a meeting call or timed asynchronous discussion thread). In some cases, an RFC may be approved outside of an RFC discussion using Pull Request reviews as a proxy for votes.
4. During an RFC discussion, the discussion leader will attempt to achieve consensus on the RFC proposal.
```

Kinda unclear to me what an "async meeting" means. Isn't the point of a meeting that... everyone is synchronously attending? 😅 

If the goal is to add flexibility to hold a discussion thread in lieu of a call, we should make that more explicit